contrib/init/sysvinit-debian/docker

```diff
@@ -119,9 +119,13 @@ case "$1" in
 	stop)
 		check_init
 		fail_unless_root
-		log_begin_msg "Stopping $DOCKER_DESC: $BASE"
-		start-stop-daemon --stop --pidfile "$DOCKER_SSD_PIDFILE" --retry 10
-		log_end_msg $?
+		if [ -f "$DOCKER_SSD_PIDFILE" ]; then
+			log_begin_msg "Stopping $DOCKER_DESC: $BASE"
+			start-stop-daemon --stop --pidfile "$DOCKER_SSD_PIDFILE" --retry 10
+			log_end_msg $?
+		else
+			log_begin_msg "Docker already stopped - file $DOCKER_SSD_PIDFILE not found."
```

Comment:
Is it "legal" (slash recommended practice) to use `log_begin_msg` like this without ever having a corresponding `log_end_msg`?